script/read-vault.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+username=$1
+password=$2
+subscriptionname=DTS-SHAREDSERVICES-SBOX
+vaultname=hmi-apim-kv-test
+secretname=hmi-apim-svc-test
+
+az login -u ${username} -p ${password}
```

Comment:
are these variable names ok to use - if DevOps are ok with this then please ignore the query.

Reply:
Will check with Dev ops...